common/src/main/res/values/strings.xml

```diff
@@ -256,13 +256,19 @@
     <string name="screen_orientation_option_value_landscape">landscape</string>
     <string name="get_help">Get Help</string>
     <string name="grant_permission">Grant Permission</string>
+    <string name="header_name_1">Name of the 1. header</string>
+    <string name="header_name_2">Name of the 2. header</string>
+    <string name="header_value_1">Value of the 1. header</string>
+    <string name="header_value_2">Value of the 2. header</string>
     <string name="help">Help</string>
     <string name="hide">Hide</string>
     <string name="high_accuracy_mode_channel_name">High accuracy location</string>
     <string name="high_accuracy_mode_notification_title">High accuracy (GPS) mode enabled</string>
     <string name="history">History</string>
     <string name="hold_to_reorder">Tap and hold to reorder</string>
     <string name="home_assistant_not_discover">Unable to find your\nHome Assistant instance</string>
+    <string name="http_header_settings">Additional Http Headers</string>
+    <string name="http_header_settings_summary">Enter Http header items with names and values. These header items will be added to all API calls to the Home Assistant server. These header items can be used (for example) to bypass Cloudflare SSO.</string>
```

Comment:
Its purpose is not "to bypass Cloudflare SSO", but "to authenticate against Cloudflare Zero Trust policies" (as described in the documentation)."Bypassing" is misleading because the goal is to authenticate. Also other minor suggestions.

```suggestion
    <string name="http_header_settings_summary">Enter HTTP headers with names and values. These headers will be added to all API calls to the Home Assistant server. These headers can be used (for example) to authenticate against Cloudflare Zero Trust policies.</string>
```

Reply:
Thanks! The thext is changed.